addons/core/.template-lintrc.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 module.exports = {
-  extends: ['recommended'],
+  extends: ['recommended', 'stylistic'],
   rules: {
     quotes: 'single',
-    'eol-last': 'editorconfig',
+    'self-closing-void-elements': false,
+    'block-indentation': false,
```

Comment:
Turned off the `block-indentation` rule as it seems like the `--fix` option is buggy and will sometimes crash the linter. This might better be manually handled with prettier.